controller/src/test/java/uni/miskolc/ips/ilona/navigation/controller/OntologyGenerationControllerIntegrationTest.java

```diff
@@ -44,6 +45,7 @@
 	protected static String USER;
 	protected static String PASSWORD;
 
+	@Autowired
```

Comment:
Spring provides various test runners for testing. Build own Application Context for testing and use that. So there would be no need for Autowired annotation in tests. The Application Context should be placed in the test folder so it wouldn't be part of the built jar.